docs/aci-interation.md

```diff
@@ -0,0 +1,130 @@
+# Features
+
+## Login
+
+You can login into Azure with `docker login azure`. This will open a Web browser and prompt the user to login to Azure. If the Docker CLI cannot open a browser, it will fall back to the [Azure device code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-device-code) and let you connect yourself.
+Note that the [Azure command line](https://docs.microsoft.com/en-us/cli/azure/) login is separated from the Docker CLI Azure login.
+
+## Docker contexts
+
+The Docker ACI integration is performed through the use of Docker contexts. You can create a context of type ACI, and then using this context you will be able to run classic Docker commands (like `docker run`) against ACI.
+Creating an ACI context requires an Azure subscription, a resource group and a region. 
+
+Before creating a context, you will need to `docker login azure` first so that the Docker CLI can find your Azure subscription.
+
+To create an ACI context, `docker context create aci` will prompt you for the relevant information, with the option to either create a new resource group or use an existing one.
+With a context created, you can either issue single commands against ACI, specifying the context name in the command line, like `docker --context acicontext run nginx`, or switch context so that all subsequent commands will be issued to ACI: `docker context use acicontext`.
+
+**Note:** You can have multiple ACI contexts associated each with different resource groups. This can be useful as it will act as namespaces for your containers. Actions on your containers will be restricted to your current context and resource group.
+
+## Single container
+
+You can start a single container on ACI with the `docker run` command. 
+The container is started on a container group, in the region and resource group associated with the ACI context.
+
+You can then list existing containers with `docker ps`. This will show the status of the containers, if they expose ports, and the container's IP address or domain name.
```

Comment:
```suggestion
You can then list existing containers with `docker ps`. This will show the status of the container, if it exposes ports, and the container's public IP address or domain name.
```